profiles/tests/test_gql_create_or_update_user_profile_mutation.py

```diff
@@ -347,20 +347,25 @@ def test_create_primary_email_for_an_existing_profile(verified, user_gql_client)
 
 
 @pytest.mark.parametrize("old_is_primary", (True, False))
-def test_existing_primary_email_remains_when_trying_to_set_the_same_email_as_a_primary_email(
-    old_is_primary, user_gql_client
+@pytest.mark.parametrize("verified", [None, False, True])
```

Comment:
Maybe always use tuples or always use lists in these?

Reply:
Off course. Staying with tuples.